wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -87,6 +91,69 @@
                     }
                 });
                 $('.listing tbody').disableSelection();
+                $("[data-order-handle]").on("keydown", function(e) {
+                    let keyCodes = {
+                        enter: 13,
+                        upArrow: 38,
+                        downArrow: 40,
+                        escape: 27
+                    }
+
+                    if (currentlySelected) {
+                        // We want to prevent default key actions (like scrolling) when we have an object selected
+                        e.preventDefault();
+                    }
+
+                    var children = $('.listing tbody').children();
+
+                    let moveElement = function() {
+                        var index = currentPosition + moveNTimesDirection;
+                        if (index < 0) {
+                            index = 0
+                        } else if (index > children.length - 1) {
+                            index = children.length - 1
+                        }
+                        var url = "{% url 'wagtailadmin_pages:set_page_position' '999999' %}".replace('999999', currentlySelected.id.substring(5));
+                        url += `?position=${(index)}`;
+                        let CSRFToken = $('input[name="csrfmiddlewaretoken"]', orderform).val();
+                        $.post(url, {csrfmiddlewaretoken: CSRFToken}, function(){
+                            addMessage('success', `"${$(currentlySelected).data('page-title')}" has been moved successfully.`)
```

Comment:
```suggestion
                            addMessage('success', `"${$(currentlySelected).data('page-title')}" has been moved successfully from item 3 to item 12.`)
```

Maybe the simplest approach is to make this message a bit smarter.

Reply:
Once you have it working, this should be translated - in JavaScript - but can be done as a separate issue.